tests/test_anthropic.py

```diff
@@ -12,9 +12,10 @@
 
 
 @pytest.mark.chat
-def test_messages():
+def test_messages(anthropic_model):
+    model_name = anthropic_model or "claude-3-opus-20240229"
```

Comment:
I want to avoid this style of setting a default value. also noticed the tests may call different models. Think we could change some of the test cases to use the same model. E.g. anthropic uses all `claude-3-haiku-20240307`. Google tests use `gemini-1.5-pro-latest`.

In that case, we could set the default value either with `pytest.ini` or with `conftest.py`. 
E.g. `pytest.ini` file and set the default values. e.g. 
``` 
[pytest]
addopts =
    --openai_model=gpt-3.5-turbo
    --openai_vision_model=gpt-4o
    --anthropic_model=claude-3-haiku-20240307
    --google_model=gemini-1.5-pro-latest
    --mistralai_model=mistral-tiny
    --lamini_model=meta-llama/Llama-2-7b-chat-hf
```